src/App.js

```diff
@@ -1,8 +1,89 @@
 import React, { Component } from 'react'
+import { css } from 'emotion'
+
+import DropDown from './components/DropDown'
+import SaveButton from './components/SaveButton'
+
+const grid = css`
+  display: grid;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-gap: 5px;
+  grid-template-rows: 50px 70px 70px 50px auto;
+  grid-template-areas:
+    '. goal .'
+    'date date date'
+    'hours hours hours'
+    '. save .'
+    'placeholder placeholder placeholder';
+  width: 240px;
+`
 
 class App extends Component {
+  state = {
+    days: [
+      { date: '2018-06-01', sleeplength: 8, id: 0 },
+      { date: '2018-06-01', sleeplength: 9, id: 1 },
+      { date: '2018-05-31', sleeplength: 7, id: 2 },
+    ],
+    value: 8,
+  }
+
   render() {
-    return <div />
+    return (
+      <div className={grid}>
+        <h2
+          className={css`
+            grid-area: goal;
+          `}
+        >
+          {' '}
+          goal: 8 hours{' '}
+        </h2>
+        <form
+          className={css`
+            grid-area: date;
+          `}
+        >
+          <label> Add night: </label>
+          <input type="date" placeholder="chose date" />
+        </form>
+        <DropDown
+          onChange={e => this.props.handleChange()}
+          onSubmit={e => this.props.handleSubmit()}
+          className={css`
+            grid-area: hours;
+          `}
+        />
+        <SaveButton
+          onClick={e => this.props.onCompare()}
+          className={css`
```

Comment:
Auch hier: weil es eine eigene Component ist, kannst du die Styles nicht per `className` anhängen. Das ginge nur, wenn du sie dann innerhalb der Component als `this.props.className` entgegen nimmst du in die richtige Stelle einsetzt. Einfachere Alternative: die `grid-area` dort in `SaveButton.js` hinzufügen, wo du auch die anderen Styles vergibst.

Reply:
ok